hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -243,7 +243,7 @@ object DataSourceWriteOptions {
    * Payload class used. Override this, if you like to roll your own merge logic, when upserting/inserting.
```

Comment:
Should we rename `DataSourceOptions.TABLE_NAME` to `TBL_NAME` to keep consistency with `HoodieWriteConfig.TBL_NAME` ?

Reply:
HoodieTableConfig.NAME is what its called in another place. Most users set it via DataSourceOptions? So should be fine leaving it more descriptive like that TABLE_NAME

Reply:
Yeah don't have a strong opinion on it. We can leave it more descriptive.